casadm/cas_main.c

```diff
@@ -741,6 +752,18 @@ int set_param_seq_cutoff_handle_option(char *opt, const char **arg)
 			cas_printf(LOG_ERR, "Error: Invalid policy name.\n");
 			return FAILURE;
 		}
+	} else if (!strcmp(opt, "promotion-threshold")) {
+		if (validate_str_num(arg[0], "sequential cutoff promotion threshold", 0,
+				4194181) == FAILURE)
```

Comment:
did you mean 4194303 (2^(32 - 10) - 1) ?

Reply:
actually we could allow 2^32-1 if we didn't convert it to KiBs in the following lines

Reply:
N/A
Btw this number (which is also cutoff threshold limit) was historically introduced as an easter egg. It's the largest prime number smaller than 2^32.